tests/PgSqlServerTest/PgSqlServerTest.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+# vim: set ai et ts=4 sw=4:
+
+import os
+import random
+import subprocess
+import requests
+import logging
+import time
+import socket
+import sys
+import psycopg2
+
+START = os.environ.get('HURMADB_PORT') is None
+PORT = int(os.getenv('HURMADB_PORT', 5432))
+con = None
+logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
+
+def hurmadb_start(port):
+    """
+    Start HurmaDB on given port, return an object that corresponds to the created process.
+    """
+    if START:
+        return subprocess.Popen(['../../hurmadb', str(port)])
```

Comment:
Database should start on random ports for both HTTP and PgSQL (even in tests for a single protocol) since if server terminates with an error port could be not available for bind()'ing for some time.